xsd/netex_framework/netex_reusableComponents/netex_vehicleType_version.xsd

```diff
@@ -194,20 +194,20 @@ Rail transport, Roads and Road transport
 					<xsd:documentation>Whether vehicles of the type are self-propelled.</xsd:documentation>
 				</xsd:annotation>
 			</xsd:element>
-			<xsd:element name="PropulsionType" type="PropulsionTypeEnumeration" minOccurs="0">
+			<xsd:element name="PropulsionType" type="PropulsionTypeEnumeration" minOccurs="0" maxOccurs="unbounded">
```

Comment:
we could have used a list of enumeration (if not we may need a wrapper tag)

        _<xsd:simpleType name="PropulsionTypeListOfEnumerations">
		<xsd:annotation>
			<xsd:documentation>List of Allowed values for propulsion type.</xsd:documentation>
		</xsd:annotation>
		<xsd:list itemType="PropulsionTypeEnumeration"/>
	</xsd:simpleType>_

Reply:
done